mocking/filemock.go

```diff
@@ -62,6 +63,12 @@ func (f *FileWrapper) Read(p []byte) (int, error) {
 	return f.file.Read(p)
 }
 
+// TODO: fix this docstring according to new documentation standard
```

Comment:
There is already documentation underneath this TODO, so what is there still to do?